Objects/Converters/ConverterRevit/ConverterRevitShared/ConverterRevit.cs

```diff
@@ -190,6 +190,17 @@ private string GetElemInfo(object o)
 
     public object ConvertToNative(Base @object)
     {
+      // schema check
+      var speckleSchema = @object["@SpeckleSchema"] as Base;
+      if (speckleSchema != null) 
+      {
+        // find self referential prop and set value to @object if it is null (happens when sent from gh)
+        var objProp = speckleSchema.GetInstanceMembers().Where(o => !o.GetType().IsEnum).Where(o => !Utilities.IsSimpleType(o.GetType())).First();
```

Comment:
Wouldn't it be safer to look for an InstanceMember that _has the same type as @object_ and that _is null?_